docs/source/reference/envvars.rst

```diff
@@ -278,3 +278,15 @@ Threading Control
 
    *Default value:* The number of CPU cores on the system as determined at run
    time, this can be accessed via ``numba.config.NUMBA_DEFAULT_NUM_THREADS``.
+
+.. envvar:: NUMBA_THREADING_LAYER
+
+   This environment variable controls the library used for concurrent execution
+   for the CPU parallel targets (``@vectorize(target='parallel')``, 
+   ``@guvectorize(target='parallel')``  and ``@njit(parallel=True)``). The 
+   variable type is string and by default is ``workqueue``, the Numba internal
+   threading library. Other valid values are:
+
+   * ``tbbpool`` - A threading layer backed by Intel TBB.
+   * ``omppool`` - A threading layer backed by OpenMP.
```

Comment:
I suggest make it to respond to just `tbb`/`TBB` and `omp`/`OMP` like in the threading selection logic. However, in the testing, it relies on the full name

Reply:
Good idea, we can fix that up.